src/views/contract/OperationsTab.vue

```diff
@@ -4,7 +4,7 @@
     <v-row>
       <v-col cols="9">
         <v-row v-if="searchable">
-          <v-col cols="12" class="pt-0">
```

Comment:
change `px-8 py-4` on `pa-8 ma-0` in `v-container` tag in the second line

Reply:
Done